gc_windows.go

```diff
@@ -0,0 +1,12 @@
+//go:build windows
+
+package main
+
+import (
+	"context"
+	"errors"
+)
+
+func (nd *Node) periodicGC(ctx context.Context, threshold float64) error {
+	return errors.New("feature not implemented on windows")
```

Comment:
IIUC the main issue here was calculating free space on Windows. What about using https://github.com/shirou/gopsutil/? It seems like it has the tools to do this generically.